src/components/input/SelectNext.tsx

```diff
@@ -110,7 +110,11 @@ function SelectMain<T>({
   useKeyPress(['Escape'], closeListbox);
 
   // Vertical arrow key for options in the listbox
-  useArrowKeyNavigation(wrapperRef, { horizontal: false, loop: false });
+  useArrowKeyNavigation(wrapperRef, {
+    horizontal: false,
+    loop: false,
+    selector: '[role="option"]',
+  });
```

Comment:
My initial intention was to change the containerRef, and use one for the listbox. This way the toggle button gets automatically excluded without having to change the selector.

However, this causes no items in the listbox to ever get focused, until you click on it. Then it becomes tab-focusable and then you can use arrow-key navigation.

I wasn't able to figure out why, so this is like the second best option.

We can leave this in the oven for a day and see if I get some inspiration to figure the other issue out and not having to mess-up with the hook internals.